src/Features/Core/Portable/ExtractMethod/MethodExtractor.Analyzer.cs

```diff
@@ -104,7 +104,9 @@ protected Analyzer(SelectionResult selectionResult, CancellationToken cancellati
                 var typeParametersInDeclaration = GetMethodTypeParametersInDeclaration(returnType, sortedMap);
 
                 // check various error cases
-                var operationStatus = GetOperationStatus(model, symbolMap, parameters, unsafeAddressTakenUsed, returnTypeHasAnonymousType);
+                var operationStatus = dataFlowAnalysisData.Succeeded
```

Comment:
when succeeded failed, I would go and actually figure out why it is failed and give users exact message how to either make it work, what specifically not supported in extract method.

otherwise, extract method just become try and error magic feature no one knows when it will work and when it will not.